eng/scripts/Language-Settings.ps1

```diff
@@ -23,17 +25,38 @@ function Get-AllPackageInfoFromRepo($serviceDirectory)
   {
     if (!$projectOutput) { continue }
 
-    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
-
+    $pkgPath, $serviceDirectory, $pkgName, $pkgVersion, $sdkType, $isNewSdk, $dllFolder = $projectOutput.Split(' ',[System.StringSplitOptions]::RemoveEmptyEntries).Trim("'")
     if(!(Test-Path $pkgPath)) {
       Write-Host "Parsed package path `$pkgPath` does not exist so skipping the package line '$projectOutput'."
       continue
     }
-    $pkgProp = [PackageProps]::new($pkgName, $pkgVersion, $pkgPath, $serviceDirectory)
+    $pkgProp = [PackageProps]::new($pkgName, $pkgVersion, $pkgPath, $serviceDirectory, $namespaces)
     $pkgProp.SdkType = $sdkType
     $pkgProp.IsNewSdk = ($isNewSdk -eq 'true')
     $pkgProp.ArtifactName = $pkgName
 
+    # Add a step to extract namespaces
+    if (!(Test-Path $dllFolder)) {
+      $dllFolder = (Split-Path $dllFolder -Parent) + "/Release"
+    }
+    if (Test-Path $dllFolder) {
+      $dllFilePaths = Get-ChildItem "$dllFolder/*" -Filter "$pkgName*" -Include *.dll -Recurse
+      if (!$dllFilePaths) {
+        Write-Warning "No dll file found under $dllFolder. Please check 'Build and Package' step for the missing."
```

Comment:
Let's not warn about this as there will be lots of valid cases where we don't have the dll build before calling this function and we don't want it be a warning.